.pre-commit-config.yaml

```diff
@@ -39,25 +40,15 @@ repos:
       - id: flake8
         name: flake8 - pd-migrator
         files: ^tools/pagerduty-migrator
-        # Make sure config is compatible with black
-        # https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
-        args:
-          [
-            --max-line-length=88,
-            "--select=C,E,F,W,B,B950",
-            "--extend-ignore=E203,E501",
-          ]
+        args: [--config=engine/tox.ini]
+        additional_dependencies:
+          - flake8-tidy-imports
       - id: flake8
         name: flake8 - dev/scripts
         files: ^dev/scripts
-        # Make sure config is compatible with black
-        # https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
-        args:
-          [
-            --max-line-length=88,
-            "--select=C,E,F,W,B,B950",
-            "--extend-ignore=E203,E501",
-          ]
```

Comment:
use consistent formatting across all Python code. Simply use the `engine/tox.ini` for this configuration